xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamPVRBase.cpp

```diff
@@ -344,6 +344,9 @@ void CInputStreamPVRBase::UpdateStreamMap()
       dStream = std::make_shared<CDemuxStream>();
 
     dStream->codec = (AVCodecID)stream.iCodecId;
+    if (dStream->codec == AV_CODEC_ID_MPEG2VIDEO)
```

Comment:
PAR changes may also happen for H.264, see [here](https://www.dvbviewer.tv/forum/topic/60058-formatumschaltung-34-169-und-umgekehrt-bei-h264-sd/?tab=comments#comment-464406) and [here](https://www.dvbviewer.tv/forum/topic/58152-formatumschaltung-funktioniert-bei-iptv-nicht/?tab=comments#comment-446029).